lib/src/main/res/values/colors.xml

```diff
@@ -2,5 +2,5 @@
 <resources>
     <color name="dayPressed">#303F9F</color>
     <color name="daySelected">#3F51B5</color>
-    <color name="dayDeselected">@android:color/transparent</color>
+    <color name="dayDeselected">@android:color/darker_gray</color>
```

Comment:
Quick fix for an issue that slipped in. Noticed this while testing the build artifacts.